pkg/operator/controller/controller.go

```diff
@@ -600,3 +576,88 @@ func deploymentConfigChanged(current, expected *appsv1.Deployment) (bool, *appsv
 	updated.Spec.Replicas = &replicas
 	return true, updated
 }
+
+// shouldPublishRouterCA checks if some ClusterIngress uses the default
+// certificate, in which case the CA certificate needs to be published.
+func shouldPublishRouterCA(ingresses []ingressv1alpha1.ClusterIngress) bool {
+	for _, ci := range ingresses {
+		if ci.Spec.DefaultCertificateSecret == nil {
+			return true
+		}
+	}
+	return false
+}
+
+// ensureRouterCAIsPublished ensures a configmap exists with the CA certificate
+// in a well known namespace.
+func (r *reconciler) ensureRouterCAIsPublished() error {
+	secret, err := r.ensureRouterCACertificateSecret()
+	if err != nil {
+		return fmt.Errorf("failed to get CA secret: %v", err)
+	}
+
+	cm := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      caCertConfigMapName,
+			Namespace: r.Namespace,
```

Comment:
Shouldn't this be in `openshift-config-managed`?

Reply:
Fixed.